### PR TITLE
Updated Second Part of Mobile Responsive Project Under Branch 'sidebranch' and Updated Linter Files

### PR DESCRIPTION
**Pull request by jkanyi-web**
Updated linter files
Updated index.html with 'My Recent Works' part of the Responsive-Mobile project 
Updated style.css with 'My Recent Works' part of the Responsive-Mobile project

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -68,24 +71,58 @@ <h2 class="supportingtext">I'm a software developer.</h2>
...
+            <img src="/Images/Vector.png" alt="straight-line">
+            <div class="placeholder-image"></div>
+
+            <h2>Multi-Post Stories</h2>
+            <p>
+                A daily selection of privately personalized
+                reads; no accounts or sign-ups required.
+                Has been the industry's standard dummy text ever since the 1500s,
+                when an unknown printer took a standard dummy text.
+            </p>
```
Comment (Gopxfs) [inline index.html]: Please take into account that you should add the "My Recent Works" + "About me" sections from [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471&t=w6f1TS1ZWFxWbNt0-0) 😉. Your project currently doesn't contain any of them, I can only see the works `section` header. Kindly make sure to add the requested sections to this branch 😄:

## My Recent Works:
### Figma Design
It should contain the first project:
image

Followed by 6 project cards:
image

### Your design
Currently, you have only a part of the first project:
image
image

Comment (Gopxfs) [inline index.html]: ## About me
### Figma design
It should contain the following introduction:
image

And a list of your abilities, as displayed below:
image

### Your design
Currently, there is no about me section on your project.

**Review by Gopxfs — CHANGES_REQUESTED** [review]
# ♻️ STATUS: REQUIRED CHANGES ♻️

## Hi 👋,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 🗒️  
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Gopxfs in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by jkanyi-web** [comment]
Hi Gabriel.
Thanks for the feedback, that part was confusing on what we we're to
actually submit.
It's just that a fellow student said that her project was accepted without
the other sections.
I'll work on it straight away.

Thanks and regards.

**Review by emmiiorji — APPROVED** [review]
Hi @jkanyi-web, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Congrats

## Good Highlights✔️
- Linter checks are passing
- Good commit messages
- Required change has been implemented


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by jkanyi-web** [comment]
Hi Emmanuel.
Thank you so much for the positive response and may you have a blessed day
and week ahead.

Regards and God bless.

**Pull request merged**